src/doc/rustdoc/src/documentation-tests.md

```diff
@@ -253,6 +253,19 @@ conversion, so type inference fails because the type is not unique. Please note
 that you must write the `(())` in one sequence without intermediate whitespace
 so that rustdoc understands you want an implicit `Result`-returning function.
 
+As of version 1.37.0, this simplification also works with `Option`s, which can
+be handy to test e.g. iterators or checked arithmetic, for example:
+
+```ignore
```

Comment:
I think you'll have to double the number of backquotes in here, otherwise your codeblock might ends up earlier than expected.

Reply:
I copied this from the codeblock above and modified it, so it appears to work nonetheless.

Should I double the number of backquotes in all code examples involving doctests or just this instance?

Reply:
No, if you tested and there is no issue, it's fine.